disnake/message.py

```diff
@@ -307,6 +313,7 @@ class Attachment(Hashable):
 
     def __init__(self, *, data: AttachmentPayload, state: ConnectionState) -> None:
         self.id: int = int(data["id"])
+        self.title: Optional[str] = data.get("title")
```

Comment:
this should also be added to `to_dict()` for consistency

Reply:
Even though it's never expected to be received by the API? This is something that you can only get from the API afaik

Reply:
You're not wrong. Since it also has `content_type` and `proxy_url` (both of which the API already doesn't expect and just discards), it would be good to be consistent, though